source/neuropod/bindings/c/np_tensor.cc

```diff
@@ -18,6 +18,15 @@ limitations under the License.
 #include "neuropod/bindings/c/np_tensor_internal.h"
 #include "neuropod/internal/neuropod_tensor_raw_data_access.hh"
 
+void NP_GetDims(const NP_NeuropodTensor *tensor, size_t *num_dims, const int64_t **dims)
+{
+    // get_dims returnis reference to internal tensor's dims collection.
+    // Use pointer to its data that is valid as long as tensor is valid.
+    auto &dims_collection = tensor->tensor->as_tensor()->get_dims();
```

Comment:
nit: explicity mark things as `const` if you don't intend to modify them (even though in this case `auto &` will deduce it)

Reply:
Yes, get_dims may have const and non-const version. Will add "const auto&"